### DISCUSSION TITLE
Is it possible to play a media from an specific second?

### DISCUSSION THREAD

**Discussion by spike886** [music-assistant/support#444] [Q&A] [answered]
I want to do an automation that reproduce 10 seconds of a song. I know that I can create an audio file with just those 10 seconds, but I want to know if it's possible to do that dynamically. 
So I can play the song, wait 10 seconds and stop it, but it needs to start from the middle of the song and not just from the beginning. Is that possible?

**Comment by marcelveldt** [comment, marked as answer]
You can use the 'media_seek' service for this:
https://www.home-assistant.io/integrations/media_player/
